app/components/BusinessGrid.tsx

```diff
@@ -47,7 +47,7 @@ export const BusinessGrid: FunctionComponent = () => {
     const NUM_DISPLAYED_CARDS = isOnSearch ? 5 : 6;
 
     return (
-        <div>
+        <div className="">
```

Comment:
This can just be an empty tag since there's no className applied here
```suggestion
        <>
```